.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
 - 8
+- 10
```

Comment:
I presume the double entry here is accidental? Should be just `12`?

Reply:
I talked to christian about it a few weeks ago and he said just to build it on 10 and 12 you are right though I should remove 8